examples/fitness/lib/measurement.dart

```diff
@@ -89,12 +89,12 @@ class MeasurementDateDialogState extends State<MeasurementDateDialog> {
       actions: [
         new FlatButton(
           child: new Text('CANCEL'),
-          onPressed: config.navigator.pop
+          onPressed: navigator.pop
         ),
         new FlatButton(
           child: new Text('OK'),
           onPressed: () {
-            config.navigator.pop(_selectedDate);
```

Comment:
Is it better practice to get the navigator during the build, or get the navigator in the handler?